modules/guides/pages/transactions.adoc

```diff
@@ -0,0 +1,603 @@
+= Couchbase Transactions
+:page-topic-type: guide
+:imagesdir: ../assets/images
+:tabs:
+:description: How to create Couchbase transactions using N1QL.
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+[abstract]
+{description}
+
+== Introduction
+
+Couchbase transactions enable you to carry out ACID (atomic, consistent, isolated, and durable) actions on the database.
+This how-to guide covers N1QL support for Couchbase transactions.
+Some SDKs also support Couchbase transactions.
+Refer to <<related-links>> for further details.
+
+Only DML (data modification language) statements are permitted within a transaction:
+xref:n1ql:n1ql-language-reference/insert.adoc[INSERT],
+xref:n1ql:n1ql-language-reference/upsert.adoc[UPSERT],
+xref:n1ql:n1ql-language-reference/delete.adoc[DELETE],
+xref:n1ql:n1ql-language-reference/update.adoc[UPDATE],
+xref:n1ql:n1ql-language-reference/merge.adoc[MERGE],
+xref:n1ql:n1ql-language-reference/select.adoc[SELECT],
+xref:n1ql:n1ql-language-reference/execfunction.adoc[EXECUTE FUNCTION],
+xref:n1ql:n1ql-language-reference/prepare.adoc[PREPARE], or
+xref:n1ql:n1ql-language-reference/execute.adoc[EXECUTE].
+
+include::partial$before-you-begin.adoc[tag=body]
+include::partial$query-tools.adoc[tags=body;!thumbs]
+
+include::partial$data-warning.adoc[]
+
+[[settings]]
+== Transaction Parameters
+:lead: The following settings set the transaction parameters for the examples in the <<multiple-statement>> section below.
+
+You can specify various settings and parameters to control how transactions work.
+You can access transaction settings and parameters through the Query Workbench, the cbq shell, or the Query REST API.
+
+[tabs]
+====
+Query Workbench::
++
+--
+To specify parameters for a Couchbase transaction, use the query run-time preferences window.
+
+. Click the cog icon icon:cog[] to display the Run-Time Preferences window.
+
+. To specify the transaction scan consistency, open the *Scan Consistency* drop-down list and select an option.
+
+. To specify the transaction timeout, enter a value in seconds in the *Transaction Timeout* box.
+
+. To specify any other parameters, click the btn:[+] button in the *Named Parameters* section.
```

Comment:
The output of this looks like `[ + ]`, should it be formatted like a button (e.g. with a border instead of the square brackets?)

Reply:
This is the default formatting that AsciiDoctor provides for a button. I think you're not the first to comment that it looks a bit strange, especially for buttons which have a symbol as the label. You could raise a style and tone issue. I think the main consideration is that it should not look like a keyboard shortcut (which is also formatted with a border). My personal preference would be just bold, with no square brackets.